### PR TITLE
7z: remove spaces after options and fix characters

### PR DESCRIPTION
**Pull request by andrik**
- [x] The PR title conforms to the recommended [templates](/tldr-pages/tldr/blob/master/CONTRIBUTING.md#commit-message).
- [x] The page follows the [content guidelines](/tldr-pages/tldr/blob/master/CONTRIBUTING.md#guidelines).

### CODE REVIEW

**Comment by mebeim** [comment]
Thanks for the PR @andrik, we usually use spaces after options though, so I'd suggest adding a space on the example which is missing it rather than removing it from the other examples.

**Comment by andrik** [comment]
Hi @mebeim, I forgot to mention that this PR refers to #2912

**Comment by mebeim** [comment]
Thanks for the link @andrik, I missed those comments, my bad. I just noticed that the man page doesn't use spaces, makes sense to be consistent then.

File: pages.pt-BR/common/7z.md [hunk 1]
```diff
@@ -5,23 +5,23 @@
 
 - Compactar um arquivo ou diretório:
 
-`7z a {{arquivo_compactado.7z}} {{caminho/arquivo_ou_diretorio}}`
+`7z a {{arquivo_compactado.7z}} {{caminho/arquivo_ou_diretório}}`
 
 - Criptografar um arquivo existente (incluindo cabeçalhos):
 
-`7z a {{arquivo_criptografado.7z}} -p {{senha}} -mhe {{arquivo.7z}}`
+`7z a {{arquivo_criptografado.7z}} -p{{senha}} -mhe {{arquivo.7z}}`
```
Comment (waldyrious) [inline pages.pt-BR/common/7z.md]: Maybe here it makes sense to split -mh from -e, and remove the space between -e and the filename? Or is it unrelated?

Comment (sbrl) [reply pages.pt-BR/common/7z.md]: The man page isn't clear, but I think that `-mhe` is all one option.

Comment (waldyrious) [reply pages.pt-BR/common/7z.md]: Wow, really? These pages seem to have quite cryptic interface :/

Comment (mebeim) [reply pages.pt-BR/common/7z.md]: `-mhe` is the `-m` option with parameter `he`, as documented in the man page: 

```-m{Parameters}
-mhe=on|off
7z format only : enables or disables archive header encryption (Default : off)```

Comment (andrik) [reply pages.pt-BR/common/7z.md]: Hey guys, I found this:

> he=[off | on]
> Enables or disables archive header encryption. The default mode is he=off.

I looked the file (/usr/share/doc/p7zip/DOC/MANUAL/cmdline/switches/method.htm) suggested on manual.

Comment (waldyrious) [reply pages.pt-BR/common/7z.md]: I'm tempted to suggest `-m"he"` or `-m'he'`, just to be sure this is properly understood. I suppose that'd work, right?

Comment (mebeim) [reply pages.pt-BR/common/7z.md]: It's the same for me really, I don't have a preference, both look clear enough.

Comment (waldyrious) [reply pages.pt-BR/common/7z.md]: @andrik what do you think?

Comment (andrik) [reply pages.pt-BR/common/7z.md]: I think it's better using -mhe=on

Comment (andrik) [reply pages.pt-BR/common/7z.md]: May I change?

Comment (waldyrious) [reply pages.pt-BR/common/7z.md]: > May I change?

No objections to the change were raised, so please go ahead :)

Comment (andrik) [reply pages.pt-BR/common/7z.md]: Done

**Pull request merged**